### DISCUSSION TITLE
python - Dr. Jun Xia

### DISCUSSION THREAD

**Discussion by giscus** [Jungeson/jungeson.github.io#4] [Announcements]
# python - Dr. Jun Xia

  双击.py文件运行python脚本文件时，命令窗口闪了一下就退出了怎么办？  可以直接添加以下命令，强制开启命令窗口:   import os    os.system(“pause”) 

http://staff.ustc.edu.cn/~xiajun/posts/2022/05/22/1/

**Comment by lijh0417** [comment]
学嘤语~o( =∩ω∩= )m
